.github/workflows/ci.yml

```diff
@@ -235,6 +244,31 @@ jobs:
           nc -z -v 127.0.0.1 9944
           chmod +x ${{ matrix.example }}
           ./${{ matrix.example }}
+  wasm_examples:
```

Comment:
I would add a newline before this, or even put this block below the next block "merge", as the merge is using only the examples. I'm almost wondering if we should put the wasm-stuff into its own yml-file (?). But I leave this up to you

Reply:
I added the newline. The merge is actually also for the wasm-example as this can also be downloaded. I also updated the merge to reflect this.